docs/userGuide/syntax/questions.mbdf

```diff
@@ -1,130 +1,325 @@
+    correctly by the SUT.
+    This is because most SUTs return an error message upon encountering the first invalid input.
+    </template>
+  </q-option>
+  <q-option>
+    It is ok to combine valid values for different inputs.
+  </q-option>
+  <q-option>
+    No more than one invalid test input should be in a given test case.
+  </q-option>
+  <q-option>
+    Each valid test input should appear at least once in a test case that doesn’t have any invalid inputs.
+  </q-option>
+  <template slot="hint">
+    How do you figure out which inputs are wrong? (or correct)
+  </template>
+</question>
+<question type="checkbox" header="Which of the following is true?">
+  <q-option correct reason="lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum">
+    1 + 1 = 2
+  </q-option>
+  <q-option reason="Division by zero is <md>**undefined**!">
+    10 / 0 = 10
+  </q-option>
+  <q-option correct>
+  11 / 11 = 1
+  </q-option>
+  <template slot="hint">
 
+  Use your calculator! :fas-calculator:
 
+  </template>
+</question>
+<question type="text" keywords="hawaii" threshold="0.5">
+  <template slot="header">Which country did the Hawaiian pizza originate from?</template>
+  <template slot="hint">Watch some pizza commercials!</template>
+  <template slot="answer">Hawaii</template>
+</question>
+</variable>
+</quiz>
+</div>
+
+<br>
+
+****Quiz Options and Slots****
+Name | Type | Default | Description
+--- | --- | --- | ---
+intro | `String` | `''` | Quiz intro markup above the question count.
+intro | Slot | `Click start to begin` | Quiz intro markup. Overrides the `intro` attribute if both are present.
+
+<!-- Included in syntax cheat sheet -->
 <span id="short" class="d-none">
 
-```html
-<question>
-  The question body
-  <div slot="hint">
-    Question hint.
-  </div>
-  <div slot="answer">
-    Question answer.
-  </div>
+```html { heading="Mcq and checkbox questions" }
+<!-- use type="mcq" for mcq questions -->
+<question type="checkbox" header="Which of the following is true?">
+  <q-option correct>
+    1 + 1 = 2
+    <template slot="reason">
+      lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum
+    </template>
+  </q-option>
+  <q-option reason="Division by zero is **undefined**!">
+    10 / 0 = 10
+  </q-option>
+  <q-option correct>
+  11 / 11 = 1
+  </q-option>
+  <template slot="hint">
+
+  Use your calculator! :fas-calculator:
+
+  </template>
 </question>
 ```
+
+```html { heading="Text questions" }
+<question type="text" header="Which country did the Hawaiian pizza originate from?"
+          keywords="hawaii" fuzzy="1" threshold="0.5" answer="It originated from Hawaii!">
+  <template slot="hint">
+
+  Watch some pizza commercials!
+
+  :pizza: :pizza: :pizza: :pizza: :pizza: :pizza: :pizza:
+
+  </template>
+</question>
+```
+
+```html { heading="Quiz" }
+<quiz>
+<question type="mcq">...</question>
+<question type="checkbox">...</question>
+<question type="text">...</question>
+</quiz>
+```
```

Comment:
Questions used as examples earlier can be reused here via `<include>`?

I'm not sure the current slide-in animation is a good fit. In this page itself, it goes over the site-nav menu -- feels like the question is overstepping its play area.

Reply:
> I'm not sure the current slide-in animation is a good fit. In this page itself, it goes over the site-nav menu -- feels like the question is overstepping its play area.

fixed, it's a bug. It's supposed to only slide in from as far as the root container goes.